submissions/2025-03-05-OpenFF-Protein-PDB-4mer-v1.0/README.md

```diff
@@ -0,0 +1,91 @@
+# OpenFF Protein PDB 4-mers V1.0
+
+### Description
+
+This dataset is composed of 1,000 4-mer structures which were extracted from PDB entries. The purpose of this dataset is to fill in gaps within the existing protein training data with secondary structures which occur in real proteins. For all scripts used to generated the input files please visit https://github.com/ajfriedman22/4mer_Generation.
+
+### General Information
+
+<!-- copy outputs from generation notebook into these fields -->
+
+- Date: 2025-03-05
+- Class: OpenFF Optimization Dataset
+- Purpose:Optimizations of 4-mers extracted from the PDB
+- Collection: OptimizationDataset
+- Name: 2025-03-05-OpenFF-Protein-PDB-4mer-v1.0
```

Comment:
This should be "OpenFF Protein PDB 4-mers v1.0" as defined in the metadata. This is the string that people will use to access the dataset from QCPortal.